packages/core/src/ScrollTo/Horizontal/HorizontalScrollListItem/HorizontalScrollListItem.styles.tsx

```diff
@@ -7,51 +7,44 @@ const name = "HvHorizontalScrollListItem";
 
 export const { staticClasses, useClasses } = createClasses(name, {
   root: {
-    padding: "10px 0",
+    padding: theme.spacing("xs", 0),
+    maxWidth: 120,
   },
   button: {
     display: "flex",
+    flexDirection: "column",
     justifyContent: "center",
     alignItems: "center",
-    height: "48px",
-    cursor: "pointer",
-    borderBottom: "none",
-    "&:hover": {
-      backgroundColor: "transparent",
-
-      "& $notSelected": {
-        height: "6px",
-        width: "6px",
-        backgroundColor: theme.colors.secondary,
-      },
-
-      "& $notSelectedRoot": {
-        backgroundColor: theme.colors.containerBackgroundHover,
-      },
-    },
-    "&:focus": {
-      outline: "none",
-    },
     "&:focus-visible": {
       ...outlineStyles,
     },
   },
   text: {
+    padding: theme.spacing("xs", "xs", "0"),
```

Comment:
The tooltip is always shown even if the label isn't truncated. Changing this to margin fixes the issue.

Reply:
That's right, I changed it to `margin` because of this, but reverted it somehow 😅 

https://github.com/lumada-design/hv-uikit-react/commit/f0a38981b799885622e616cca9dcdb7270065391